app/test_app/functional/api/test_user_api.py

```diff
@@ -79,12 +82,17 @@ def test_create_user(test_client, authed_sempo_admin_user, init_database, create
         assert data['user']['is_vendor'] is False
         assert data['user']['is_tokenagent'] is False
         assert data['user']['is_groupaccount'] is False
-        assert data['user']['transfer_accounts'][0]['balance'] == 0
         assert data['user']['location'] == 'Elwood'
         assert data['user']['business_usage_id'] == init_database.session.query(TransferUsage)\
             .filter_by(name=business_usage_name).first().id
         assert data['user']['referred_by'] == user_phone_accessor(create_transfer_account_user)
 
+        if initial_disbursement is not None:
+            assert data['user']['transfer_accounts'][0]['balance'] == initial_disbursement
+        else:
+            db_user = init_database.session.query(User).get(data['user']['id'])
+            assert data['user']['transfer_accounts'][0]['balance'] == db_user.default_organisation.default_disbursement
```

Comment:
adds a test for both usecase's when a user with default disbursement is created and when a custom disbursement is created (in this case, the amount of zero)